Source/Core/VideoBackends/D3D12/D3D12Gfx.cpp

```diff
@@ -43,98 +41,75 @@ Renderer::Renderer(std::unique_ptr<SwapChain> swap_chain, float backbuffer_scale
+                                                            AbstractTexture* depth_attachment)
 {
   return DXFramebuffer::Create(static_cast<DXTexture*>(color_attachment),
                                static_cast<DXTexture*>(depth_attachment));
 }
 
 std::unique_ptr<AbstractShader>
-Renderer::CreateShaderFromSource(ShaderStage stage, std::string_view source, std::string_view name)
+Gfx::CreateShaderFromSource(ShaderStage stage, std::string_view source, std::string_view name)
 {
   return DXShader::CreateFromSource(stage, source, name);
 }
 
-std::unique_ptr<AbstractShader> Renderer::CreateShaderFromBinary(ShaderStage stage,
-                                                                 const void* data, size_t length,
-                                                                 std::string_view name)
+std::unique_ptr<AbstractShader> Gfx::CreateShaderFromBinary(ShaderStage stage, const void* data,
+                                                            size_t length, std::string_view name)
 {
   return DXShader::CreateFromBytecode(stage, DXShader::CreateByteCode(data, length), name);
 }
 
 std::unique_ptr<NativeVertexFormat>
-Renderer::CreateNativeVertexFormat(const PortableVertexDeclaration& vtx_decl)
+Gfx::CreateNativeVertexFormat(const PortableVertexDeclaration& vtx_decl)
 {
   return std::make_unique<DXVertexFormat>(vtx_decl);
 }
 
-std::unique_ptr<AbstractPipeline> Renderer::CreatePipeline(const AbstractPipelineConfig& config,
-                                                           const void* cache_data,
-                                                           size_t cache_data_length)
+std::unique_ptr<AbstractPipeline> Gfx::CreatePipeline(const AbstractPipelineConfig& config,
+                                                      const void* cache_data,
+                                                      size_t cache_data_length)
 {
   return DXPipeline::Create(config, cache_data, cache_data_length);
 }
 
-std::unique_ptr<BoundingBox> Renderer::CreateBoundingBox() const
-{
-  return std::make_unique<D3D12BoundingBox>();
-}
-
-void Renderer::Flush()
+void Gfx::Flush()
 {
   ExecuteCommandList(false);
 }
 
-void Renderer::WaitForGPUIdle()
+void Gfx::WaitForGPUIdle()
 {
   ExecuteCommandList(true);
 }
 
-void Renderer::ClearScreen(const MathUtil::Rectangle<int>& rc, bool color_enable, bool alpha_enable,
-                           bool z_enable, u32 color, u32 z)
+void Gfx::ClearRegion(const MathUtil::Rectangle<int>& target_rc, bool color_enable,
+                      bool alpha_enable, bool z_enable, u32 color, u32 z)
 {
   // Use a fast path without the shader if both color/alpha are enabled.
-  const bool fast_color_clear = color_enable && (alpha_enable || !EFBHasAlphaChannel());
+  const bool fast_color_clear = color_enable && alpha_enable;
   if (fast_color_clear || z_enable)
   {
-    MathUtil::Rectangle<int> native_rc = ConvertEFBRectangle(rc);
```

Comment:
I guess this was intentionally removed?

Reply:
Just noticed that `D3D` didn't have a `ClearScreen()`

Reply:
Ah, finally got to `AbstractGfx` and saw the comment :)

Reply:
It's a bit misleading that github is showing the diff between these two functions, they are different functions with different APIs. `ClearScreen` takes a rect in EFB coordinates, while the new `ClearRegion` takes a target rect in native coordinate space. 

Parts of the old `ClearScreen` (such as rectangle conversion) have been moved to shared code in `FramebufferManager::ClearEFB`.

> Just noticed that D3D didn't have a ClearScreen()

Yeah, it never had a specialised version of it, relying on the common code (now in `AsbstractGfx::ClearRegion`) for all clear operations.